src/app.rb

```diff
@@ -17,12 +17,14 @@ def initialize
 
   # List of books...
   def list_books
-    puts(@books.map { |book| "Title: \"#{book.title}\", Author \"#{book.author}\"" })
+    @books.map { |book| "Title: \"#{book.title}\", Author \"#{book.author}\"" }
```

Comment:
- [ ] Great effort working on refactoring your school library project, everything seems to work fine in your project after refactoring except the `List all books`, I noticed when trying to show all the books added by a user by clicking on the `List all book` the list of books are not showing. I would kindly suggest you work on it and be sure the app is working perfectly as expected. Thank you

Screenshot 👇 

![skull](https://github.com/Kaiserabbas/school_library/assets/78634841/319098b4-6dd7-42ee-9be9-f18ff645732b)